referrer-policy/no-referrer-policy/no-referrer-policy.subresource.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>This page reports back it's referrer URL to the parent frame</title>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body style="background:#f4f4f4;">
+    <h2>This page reports back it's referrer URL to the parent frame.</h2>
+
+    <script>
+      // Display this page's location (iframe page - subresource to parent).
+      appendPageInfo("Child frame");
+
+      var pageInfo = {
+          location:document.location.toString(),
+          referrer:document.referrer
```

Comment:
Yes, I agree, I think it would be good if I serialized all the headers via python to JS and provide them to the subresource - this can be reused in multiple tests.